src/main/java/anichan/commands/ViewWatchlistCommand.java

```diff
@@ -22,9 +22,25 @@
     protected static final String NO_WATCHLIST_ERROR = "There are no watchlists in your workspace!";
     protected static final String EMPTY_WATCHLIST_ERROR = "There are no anime in ";
     
-    private Integer watchlistIndex = 0;
+    private Integer watchlistIndex;
     private static final Logger LOGGER = AniLogger.getAniLogger(ViewWatchlistCommand.class.getName());
 
+    /**
+     * Creates a new instance of ViewWatchlistCommand.
+     */
+    public ViewWatchlistCommand() {
```

Comment:
Is this constructor still needed?

Reply:
Yes, this constructor is needed as view command now takes in the optional parameter `-v <WATCHLIST_INDEX>`, and an empty constructor is required to not assign any value to the watchlist_index in viewCommand

Reply:
Ah, I see, my mistake then.